packages/react-adminer/src/components/List.tsx

```diff
@@ -145,7 +151,14 @@ const ListChild: React.FC<Props> = ({ entityConfig, entityName, filter = true })
 			name: f.label ?? f.name,
 			title: (
 				<Box display="flex" justifyContent="space-between">
-					<Box>{f.label ?? f.name}</Box>
+					<Box>
+						{f.label ?? (
+							<>
+								<div style={{ display: 'inline-block', marginTop: '0px' }}>{f.name}</div>
```

Comment:
`f.label ?? f.name` -> když je label, použij label, jinak použij name

`{f.label ?? (..` když je label, použij label, jinak použij to druhé.

To znamená že helper dokážeš vypsat jen když neexisuje label, ale my to chcem pro oba stavy.